### ISSUE TITLE
Controller 'googleChart', required by directive 'onError', can't be found!

### ISSUE THREAD

**Issue by ilyador** [angular-google-chart/angular-google-chart#201]
I use 0.0.11 with angular 1.3.2 and I get `error: Error: [$compile:ctreq] Controller 'googleChart', required by directive 'onError', can't be found!`

**Comment by dzonatan** [comment]
This is related to the same directive/attribute names conflict issue. In your case you or your 3rd party lib have `onError` directive/attribute declared which is already declared in this google-chart lib as a directive...

The similar error appears when you try to use this lib with [ui-select](https://github.com/angular-ui/ui-select)
```
Controller 'googleChart', required by directive 'onSelect', can't be found!
```
Both **ui-select** and **angular-google-chart** declares `on-select` directive...

To fix this and avoid similar recurrences later all directive names should use **prefix** like `agc-on-select`!!!

**Comment by Balrog30** [comment]
I agree.  I think I took the project in a direction it wasn't prepared for without namespacing the directives (not a big deal when there was only one).  I think `agc` is a good namespace.

I originally made it on-select because it was previously an attribute on the directive's element, but not a directive itself. Keeping the name made it a non-breaking change, but created a naming conflict with other modules.

I was thinking of re-writing it so that your template would look like this:

``` HTML
<agc-chart chart="chartObject" agc-on-select="selectHandler(selectedItem)"></agc-chart>
```

OR

``` HTML
<div agc-chart="chartObject"></div>
```

**Comment by Balrog30** [comment]
Should be solved by #215. Main chart directive not renamed, but event directives have been name-spaced.
